packages/trace-viewer/src/ui/networkResourceDetails.css

```diff
@@ -51,6 +58,18 @@
   overflow: hidden;
 }
 
+.expandable:has(.cm-wrapper) {
```

Comment:
we would do it the other way around, say expandable-in-something instead of the reverse :has.